homework/H4_Deep_Learning.jl

```diff
@@ -54,7 +54,7 @@ html"""
 """
 
 # ╔═╡ ce8fd3c2-4fe8-4408-8479-efc81f1a4147
-md"Due date: **Monday, May 20th 2024 at 23:59**"
+md"Due date: **Thursday, November 21st 2024 at 23:59**"
```

Comment:
Why isn't Monday, November 18th the due date?

Reply:
That's because I oriented myself on the ISIS dates: 
![image](https://github.com/user-attachments/assets/8cc4ef4d-ae17-4411-a2db-48c578102480)
I can change it to the 18th instead, if you say that's correct

Reply:
Sorry, my bad for the error on ISIS! Monday, November 18th is the correct due date.

Reply:
The due date has now been updated!